packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -1,7 +1,8 @@
-// Copyright 2015 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// Copyright 2015 The Chromium Authors. All rights reserved.		
+// Use of this source code is governed by a BSD-style license that can be		
+// found in the LICENSE file.		
 
+ 
 import 'package:flutter/foundation.dart';
```

Comment:
looks like you have some accidental whitespace changes near the top (new spaces and new lines)